src/PreserveModuleNamePlugin.ts

```diff
@@ -57,11 +147,13 @@ export class PreserveModuleNamePlugin {
           if (/^async[?!]/.test(realModule.rawRequest)) 
             id = "async!" + id;
           
-          id = id.replace(/\\/g, "/");
+          id = id.replace(/\\/g, '/');
```

Comment:
All strings use double-quotes, why did you change this one to single-quote? 
Also: consistency is good, make sure your new code uses double-quotes (I spotted some single-quotes).